coder_sniffer/DrupalPractice/Sniffs/Objects/StrictSchemaDisabledSniff.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * DrupalPractice_Sniffs_Objects_StrictSchemaDisabledSniff.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+namespace DrupalPractice\Sniffs\Objects;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractScopeSniff;
+
+/**
+ * Checks that $strictConfigSchema is not set to FALSE in test classes.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+class StrictSchemaDisabledSniff extends AbstractScopeSniff
```

Comment:
Hm, are you sure AbstractScopeSniff is the correct class and not AbstractVariableSniff with processMemberVar()?

What happens if you have a variable named $strictConfigSchema as local variable in a class method? Please add a test case for that so that we see that this sniff does not trigger there.

Reply:
Good catch! That's more along the lines of the 2.x version of this I made, but I missed that base class.